tests/test_eload_validation.py

```diff
@@ -241,3 +241,38 @@ def test_mark_valid_files_and_metadata(self):
         expected = {'analyses': {'ELOAD_2_analysis_alias': {'vcf_files': ['test.vcf']}},
                     'metadata_spreadsheet': '/path/to/the/spreadsheet'}
         assert self.validation.eload_cfg.query('validation', 'valid') == expected
+
+    def test_validate_sample_names(self):
+        metadata_file = os.path.join(self.resources_folder, 'metadata_2_analysis_same_samples.xlsx')
+        self.validation.eload_cfg.set('submission', 'metadata_spreadsheet', value=metadata_file)
+        self.validation.eload_cfg.set('validation', 'sample_check', value={})
+        with patch.object(self.validation, '_get_dir', return_value=os.path.join(self.resources_folder, 'vcf_dir')):
```

Comment:
This is fine for now, but I do think that if we want to allow the eload directory to be changed by individual tests, it would be better to allow the directory path to be optionally passed into the Eload object, and just constructing it from the config by default. It would make a few of these tests simpler - kind of like how we started allowing the config to be passed in rather than always reading it from file.